soaktest/README.md

```diff
@@ -0,0 +1,28 @@
+# Information for soak test for go client
+The soak test contains 2 clients:
+soakclient.go for Process-1 : Create Producer-1 to keep sending messages to the input-topic.
+Create Consumer-2 to receive messages from the output-topic.
+
+soakclient_transaction.go for Process-2: Create Consumer-1 to receive messages from the input-topic.
+Create transactional producer Producer-2 to sending the received messages to the output-topic.
+The initTransaction API registers a transactional.id when creating the producer.
+The producer sends the messages to the output-topic.
+Commit transaction after sending 100 messages each time or 1 second.
+
+# Instruction on configuration and running
+
+# Build soaktest API
+    $ cd confluent-kafka-go/soaktest
+    $ go mod init soaktest
+    $ go mod tidy
+    $ go install soaktest
+   
+# Build and run soakclient.go
+    $ cd confluent-kafka-go/soaktest/soakclient
+    $ go build
+    $ ./soakclient [-broker] [-inputTopic] [-outTopic] [-groupId] [numPartitions] [-replicationFactor] [-ccloudAPIKey] [-ccloudAPISecret]
```

Comment:
Syntax is typically:
`$ ./soakclient -broker <broker> -inputTopic <..> ` for required arguments.
`[..]` denotes optional arguments, e.g.:
`$ ./soakclient -broker <..> -inputTopic <..> [-timeout <..>]`